src/routes.rs

```diff
@@ -58,23 +64,97 @@ pub fn catchers() -> Vec<rocket::Catcher> {
     catchers![not_found]
 }
 
-struct AuthorisedUser(String);
+struct AuthorisedUser {
+    // other useful fields could include organisation home scope permissions
+    username: String,
+    authorized: bool,
```

Comment:
I'll have to double check the docs, but I think AuthorizedUser should only be possible for a user that passes authorization (i.e. this should always be true).